lib/api/controllers/securityController.js

```diff
@@ -139,16 +159,14 @@ class SecurityController {
   searchRoles(request) {
     checkSearchPageLimit(request, this.kuzzle.config.limits.documentsFetchCount);
 
-    let
+    const
       controllers = request.input.body && request.input.body.controllers,
       from = request.input.args && request.input.args.from && Number(request.input.args.from),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 94 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)